content/docs/command-reference/exp/run.md

```diff
@@ -119,10 +136,8 @@ CPU cores).
   name will generated by default, such as `exp-f80g4` (based on the experiment's
   hash).
 
-- `-f`, `--force` - reproduce an experiment pipeline, regenerating its results,
-  even if no changes were found. For `checkpoint` experiments, `dvc exp run -f`
-  will force regeneration of an existing checkpoint experiment branch from
-  scratch.
+- `--temp` - run this experiment in a separate temporary directory (in
+  `.dvc/tmp/exp<n>`) instead of your workspace. Useful for
```

Comment:
> Useful for

Oops I forgot to finish this haha. I guess I'm not sure what the use case is here? If you're already running experiments you're probably OK with changing your workspace? You can always `git reset --hard; dvc checkout`.

Reply:
It's just to give the user the option to run individual experiments in the same execution manner as `--queue`, without polluting their workspace.

Reply:
I'm not sure that this needs to say anything beyond `run this experiment in a separate temporary directory instead of your workspace.`

There should maybe be a note somewhere else on the `exp run` page documenting where the temp dir for queued/parallel and `--temp` runs will go, but I'm not sure that kind of implementation detail needs to be documented in the usage section?

Reply:
> I'm not sure that this needs to say anything beyond

We explain motivation/uses whenever possible. But if this is just a why-not utility then yes, Idk how to explain it anyway (for now — we'll see how users adopt it later).

> There should maybe be a note somewhere...

There is 🙂
![image](https://user-images.githubusercontent.com/1477535/109741862-911df280-7b93-11eb-99c0-06a7ebe61698.png)

> not sure that kind of implementation detail needs to be documented in the usage section

It's a good question. In this case it takes up 3 words in a parenthesis so why-not.